redirect-rules/simple/rule.js

```diff
@@ -0,0 +1,36 @@
+function redirectToConsentForm (user, context, callback) {
+  var hasConsented = user.app_metadata && user.app_metadata.has_consented;
+
+  // redirect to consent form if user has not yet consented
+  if (!hasConsented && context.protocol !== 'redirect-callback') {
+    var auth0Domain = auth0.baseUrl.match(/([^:]*:\/\/)?([^\/]+\.[^\/]+)/)[2];
+
+    context.redirect = {
+      url: configuration.CONSENT_FORM_URL +
+        '&auth0_domain=' + encodeURIComponent(auth0Domain) +
```

Comment:
@tjanczuk: As I think you understand, we're passing the domain in this case from the rule to the webtask because that way the webtask can generically handle instances of this rule from any Auth0 tenant. But I can see the security concern. Perhaps I should just make note of that in the README and especially encourage the reader to explicitly configure it if they use their own instance of the webtask?